scripts/upload-apk.sh

```diff
@@ -13,28 +13,36 @@ git clone --quiet --branch=apk https://fossasia:$GITHUB_API_KEY@github.com/fossa
 
 cd apk
 
-\cp -r ../app/build/outputs/apk/*/**.apk .
-\cp -r ../app/build/outputs/apk/debug/output.json debug-output.json
-\cp -r ../app/build/outputs/apk/release/output.json release-output.json
+\cp -r ../app/build/outputs/apk/*/*/**.apk .
+\cp -r ../app/build/outputs/apk/fdroid/debug/output.json fdroid-debug-output.json
+\cp -r ../app/build/outputs/apk/playstore/debug/output.json playstore-debug-output.json
+\cp -r ../app/build/outputs/apk/fdroid/release/output.json fdroid-release-output.json
+\cp -r ../app/build/outputs/apk/playstore/release/output.json playstore-release-output.json
 \cp -r ../README.md .
```

Comment:
@CloudyPadmal this should be removed, as in apk branch the readme would not be linked to the doc images. It's better to have a seperate readme for this branch. Copying the current readme should not be done. Let me know your thoughts on it

Reply:
:+1: @neel1998 remove this copy for now. We'll create a README specific for APK branch later.